backend/test/institution_handlers_tests/institution_handler_test.py

```diff
@@ -6,8 +6,8 @@
 from models import User
 from models import Institution
 from handlers.institution_handler import InstitutionHandler
-from worker import AddAdminPermissionsInInstitutionHierarchy
-from worker import RemoveAdminPermissionsInInstitutionHierarchy
+from worker_handler import AddAdminPermissionsInInstitutionHierarchyHandler
```

Comment:
You could make these 2 imports in the same line separating them by comma

Reply:
Acredito que fica muito grande.